polymer/simple-wrapper-element/pubspec.yaml

```diff
@@ -0,0 +1,40 @@
+name: 'simple_wrapper_element'
+description: A web app that wraps a paper element inside another Polymer element.
+
+dependencies:
+  browser: ^0.10.0
+  polymer_elements: ^1.0.0
+  polymer: ^1.0.0
+  web_components: ^0.12.0
+  reflectable: ^0.2.0
+
+transformers:
+- web_components:
+    entry_points: web/index.html
+- reflectable:
+    entry_points: web/index.dart
+- $dart2js:
```

Comment:
Lets make this consistent (I don't see it in polymer/polymer_list/pubspec.yaml). Either add it there or remove here - I am a bit torn on which is better. We can chat offline.